src/decimal128.ts

```diff
@@ -796,7 +803,7 @@ export class Decimal128 {
   }
 
   inspect(): string {
-    return `Decimal128.fromString("${this.toString()}")`;
+    return `Decimal128("${this.toString()}")`;
```

Comment:
I think we should have `new` here to be consistent with the other types. Or we should make all the types not print out `new` in front of the type construction. @mongodb/dbx-node thoughts?

Reply:
@nbbeeken You’re right, thanks for pointing this out. I’ve added the `new` for consistency. If we can, let’s keep the `new`-or-not discussion separate from this ticket here. :)

Reply:
Fair enough! Thanks :)